docs/ops/memory/mem_migration.md

```diff
@@ -0,0 +1,184 @@
+
+| **Deprecated option**               | **Interpreted as**                                                                                                                                                                                                                                                                                                        |
+| :---------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| taskmanager.heap.size               | - [taskmanager.memory.flink.size](../config.html#taskmanager-memory-flink-size-1) for standalone deployment <br/> - [taskmanager.memory.process.size](../config.html#taskmanager-memory-process-size-1) for containerised deployments <br/> See also [how to migrate total memory](#total-memory-previously-heap-memory). |
+| taskmanager.memory.size             | taskmanager.memory.managed.size, see also [how to migrate managed memory](#managed-memory)                                                                                                                                                                                                                                |
+| taskmanager.network.memory.min      | [taskmanager.memory.network.min](../config.html#taskmanager-memory-network-min)                                                                                                                                                                                                                                           |
+| taskmanager.network.memory.max      | [taskmanager.memory.network.max](../config.html#taskmanager-memory-network-max)                                                                                                                                                                                                                                           |
+| taskmanager.network.memory.fraction | [taskmanager.memory.network.fraction](../config.html#taskmanager-memory-network-fraction)                                                                                                                                                                                                                                 |
+
+Although, the network memory configuration has not changed too much it is recommended to verify its configuration.
+It can change if other memory components have new sizes, e.g. the total memory which the network can be a fraction of.
+See also [new detailed memory model](mem_setup.html#detailed-memory-model).
+
+The container cut-off configuration options, ‘containerized.heap-cutoff-ratio’ and ‘containerized.heap-cutoff-min’,
```

Comment:
Do we want to also add links for these two options?

Reply:
I will add here as we say they are still valid for JM but not later where we say they are not available anymore.